src/views/Dashboard.tsx

```diff
@@ -1,12 +1,22 @@
 import React from 'react';
 import DashboardHeader from '../components/DashboardHeader';
+import SideBar from '../components/sidebar/SideBar';
 
 const Dashboard: React.FunctionComponent = () => {
     return (
-        <React.Fragment>
-            <h1>Dashboard</h1>
-            <DashboardHeader />
-        </React.Fragment>
+        <div className="container view">
```

Comment:
Should be section view to make it full width, also can you make some scss changes to make this full-height too?